### ISSUE TITLE
Error reverse engineer after picking database connection EF Core Power Tools version: Version 2.4.140.0 /2.4.150.0

### ISSUE THREAD

**Issue by dcanamares** [ErikEJ/EFCorePowerTools#455]
After choosing the database connection, I 've got an error
![imagen](https://user-images.githubusercontent.com/4290346/89896139-39ab3b80-dbdd-11ea-9ba6-3d06e963311a.png)

### Further technical details
The project is a Net Core 2.2. I 've got installe Net Core 3.1
EF Core Power Tools version: Version 2.4.140.0 or 2.4.150.0

SQLite ADO.NET Provider included: 1.0.111.0
SQLite EF6 DbProvider in GAC - No

System.Data.SQLite DDEX provider - No
SQLite Simple DDEX provider - Yes

Database engine: SQL Server

Visual Studio version: Microsoft Visual Studio Enterprise 2017 
Versión 15.9.21

Is there any way to download a july or june version?

Regards.

**Comment by ErikEJ** [comment]
Do you have the .NET Core 3.1 x86 runtime installed? https://github.com/ErikEJ/EFCorePowerTools/issues/452#issuecomment-671909781

**Comment by ErikEJ** [comment]
Duplicate of #452

**Comment by ErikEJ** [comment]
@dcanamares Let me know if you are unblocked, latest build does not require the x86 runtime any more (2.4.151) - I also have a link to an old version, but prefer to move forward.

**Comment by dcanamares** [comment]
After installing 2.4.151 It's working again. I didn't need to install x86 versions.
Thanks.
